### ISSUE TITLE
Unable to print md file with themes

### ISSUE THREAD

**Issue by sz332** [webpro/reveal-md#176]
When I try to print any markdown document using 

reveal-md.cmd slides.md --print 

it is printed correctly, however if I try to print using a theme provided my reveal-md,
it creates an empty document.

reveal-md.cmd slides.md --theme beige --print 

[slides.txt](https://github.com/webpro/reveal-md/files/1723622/slides.txt)

node --version
8.9.4
reveal-md --version
1.1.3
phantomjs --version
2.1.1

Windows7

**Comment by sz332** [comment]
Extension: the only theme that actually seems to work is black.

**Comment by almondtools** [comment]
Same behavior for same configuration, but

- theme "white" works
- custom theme works, except for background images (but this might be another problem)

**Comment by humanfactors** [comment]
I cannot get `reveal-md -w slides.md` to read the theme defined in the YAML header. I can get this to work, however, by manually overriding with ` --theme $(THEME)`. On the other hand, `reveal-md --static` works just fine.

**Comment by webpro** [comment]
Hi, been working on a rewrite, which should improve things. Could you guys please give this a shot? You can install with:

```
npm install --global reveal-md@rc
```

Let me know any issues you may encounter, and we can fix up before a final v2.

**Comment by webpro** [comment]
Can you guys please try again with version 2.0.0-rc.4? Feel free to (re)open tickets if any issues occur.

```
npm install reveal-md@rc --global
```

**Comment by DMXL** [comment]
Tested `--print` with rc.6. It seems to work just fine! @webpro

**Comment by almondtools** [comment]
Where have pictures to be placed such that they are included in the pdf?

**Comment by webpro** [comment]
@almondtools Next to the Markdown in the same directory, or a subdirectory.

**Comment by almondtools** [comment]
My custom theme gets applied, yet background images stay white. Example:

```
.reveal .backgrounds .slide-background.title.present {
  background: url("background.png") center 0/100% no-repeat; }
```
I placed `background.png` in the theme directory and in the current directory but it never got included in the pdf. I did not try other ways to include the image.

Maybe related: generating a static side with custom theme links the css (`./theme/custom.css`), yet it does not copy the theme directory to the static site directory. The html appears completely unstyled. Copying the directory manually makes it work.
